src/inpar/4C_inpar_rebalance.cpp

```diff
@@ -33,6 +33,9 @@ void Inpar::Rebalance::set_valid_parameters(Teuchos::ParameterList& list)
       "Tolerance for relative imbalance of subdomain sizes for graph partitioning of unstructured "
       "meshes read from input files.",
       &meshpartitioning);
+
+  Core::Utils::int_parameter("MIN_ELEPROC", 0,
+      "Minimum no. of elements per processor for parallel redistribution.", &meshpartitioning);
```

Comment:
Can you include the fact, that 0 disables the minimum number of elements per proc, into the documentation string?

```suggestion
  Core::Utils::int_parameter("MIN_ELEPROC", 0,
      "This parameter defines the minimum number of elements to be assigned to any MPI rank during redistribution. Use 0 to not interfere with the minimal size of a subdomain.", &meshpartitioning);
```

Reply:
```suggestion
  Core::Utils::int_parameter("MIN_ELE_PER_PROC", 0,
      "Minimum no. of elements per processor for parallel redistribution.", &meshpartitioning);
```

Reply:
Renamed.